src/common/http/platform/beast/http.cpp

```diff
@@ -1887,7 +1887,9 @@ void Server::PrepareNewStream() {
 void Server::AsyncAccept(StreamPtr stream) {
 	acceptor_.async_accept(stream->socket_, [this, stream](const error_code &ec) {
 		if (ec) {
-			log::Error("Could not accept connection: " + ec.message());
+			if (ec != errc::operation_canceled) {
```

Comment:
Q: should we add a lower level log for all instances?

Reply:
Originally I did not expect to get any errors here at all. Then I realized that this particular one makes sense, but I still don't expect to get other errors, so I think it's best to keep it.